### ISSUE TITLE
Sweet Alert modal not centered

### ISSUE THREAD

**Issue by felipemendes1994** [t4t5/sweetalert#649]
Hello Guys,

I'm tryin' to use Sweet Alerts in a new project I've been workin' on... But when I click at the button, or call the Swal function on "$(document).ready()", the swal alert appears not in middle of the screen, but at the bottom of the page.

![image](https://cloud.githubusercontent.com/assets/22458948/21928129/cde051a4-d967-11e6-8dc0-bdfe6afef00c.png)

I'have already raplaced the library, but it didn't work... Do you have any idea of what the hell may be going on?

Some code:

```
$(document).ready(function(){
	setTimeout(function() {
		swal({
		      title: "Done!",
		      text: "It's not working",
		      type: "success",
		      confirmButtonText: "Ok"
		});
	}, 500);
});
```

```
$("#btnEnviar").click(function(){
     swal({
         title: "Done!",
         text: "It's not working",
         type: "success",
         confirmButtonText: "Ok"
      });
});
```

As I said, in both cases, the alert appears at the bottom of the page, not in the middle!!

Oh yeah, I'm also using bootstrap

Thanks Folks!!

**Comment by quincarroll** [comment]
I realize that this problem is a few months old, but I just sorted a similar problem out, and thought someone might benefit from what I did to fix it.

Make sure that you have the link to sweetalert.css on your page. Someone (one of the other developers) removed it from my login page on my site, and exactly the same thing happened... The alert popped up at the bottom of the page behind the footer, and looked like it wasn't working until I scrolled down the page and found it quite by accident.
